src/app/editor/contextView.js

```diff
@@ -129,11 +130,36 @@ class ContextView {
       }
     }
 
+    function next () {
+      var currentName = node.attributes.name
+      if (currentName === this.refName) {
+        this.ref = this.ref === undefined ? 0 : this.ref
+      } else { this.ref = 0 }
+      var nodes = self._api.contextualListener._activeHighlights
```

Comment:
since `_activeHighlights` is "private" (declared with underscore), better not accessing it like this but at least create a getter in contextualListener

Reply:
Good call. Fixed that